test/jvm/src/main/scala/zio/test/RunnableSpec.scala

```diff
@@ -37,8 +38,11 @@ abstract class RunnableSpec[L, T, E, S](runner0: TestRunner[L, T, E, S])(spec0:
    * TODO: Parse command line options.
    */
   final def main(args: Array[String]): Unit = {
-    val results     = runner.unsafeRun(spec)
-    val hasFailures = results.exists { case TestCase(_, test) => test.isLeft; case _ => false }
+    val ExecutionResult(results, summary) = runner.unsafeRun(spec)
+    val hasFailures                       = results.exists { case TestCase(_, test) => test.isLeft; case _ => false }
+
+    println(summary)
```

Comment:
Can we use the `TestLogger` in the `TestRunner` here?

Reply:
Fixed. Shouldn't this be done similarly for JS?

Reply:
I just noticed that `RunnableSpec` differs between JS and JVM only by:
  - the way tests are run (sync vs async)
  - lack of reeporting and `sys.exit` on JS side.

Wouldn't it work to move `RunnableSpec` into `shared` and run differently basing on `TestPlatform`?

Reply:
Yes, that is a great idea. We introduced `TestPlatform` after we created `RunnableSpec` which is why we had to create two separate files but if we can integrate into one file that will definitely make things more maintainable in the future.